templates/torrc.j2

```diff
@@ -16,19 +16,21 @@ Log {{ log }}
 {# Maybe not render if not needed? #}
 RunAsDaemon {{ renderbool(torrc_run_as_daemon) }}
 
-{%- if torrc_data_directory %}
+{% if torrc_data_directory %}
```

Comment:
It is meaningful! Essentially this dash was removing all preceding whitespace including line breaks so that the following block was rendered at the end of the RunAsDaemon line, which caused the torrc file to be invalid